sentry-types/src/protocol/envelope.rs

```diff
@@ -152,14 +152,39 @@ impl From<SampleProfile> for EnvelopeItem {
 /// An Iterator over the items of an Envelope.
 #[derive(Clone)]
 pub struct EnvelopeItemIter<'s> {
-    inner: std::slice::Iter<'s, EnvelopeItem>,
+    inner: Option<std::slice::Iter<'s, EnvelopeItem>>,
```

Comment:
No need to make this an `Option`, you can always just create an empty slice iter.

Reply:
D'oh